baybe/constraints/base.py

```diff
@@ -104,6 +105,21 @@ def get_invalid(self, data: pd.DataFrame) -> pd.Index:
             The dataframe indices of rows where the constraint is violated.
         """
 
+    def to_polars(self) -> pl.Expr:
+        """Translate the constraint to Polars expression for filtering.
+
+        Returns:
+            The Polars expression to pass to ``polars.LazyFrame.filter``.
+
+        Raises:
+            NotImplementedError: If the constraint class does not have a Polars
+                implementation.
+        """
+        raise NotImplementedError(
+            f"The {self.__class__.__name__} constraint does not have a Polars "
```

Comment:
Thanks for the change, looking good. Perhaps minor last change: The error message would be somewhat weirdly printed e.g. as "...Constraint constraint does not..."
```suggestion
            f"'{self.__class__.__name__}' does not have a Polars "
```